components/rootless_studio/default/entrypoint.sh

```diff
@@ -15,6 +15,7 @@ case "$1" in
     hab pkg exec core/hab-plan-build hab-plan-build "$@";;
   run)
     shift
-    hab pkg exec core/hab-backline bash --login -c "$@";;
+    # shellcheck disable=SC2046
+    hab pkg exec core/hab-backline env $(load_secrets) bash --login -c "$@";;
```

Comment:
Considering that things like randomly generated keys are passed in through secrets, and may contain globbing characters, I think it's probably important to do this the safe way:
```bash
    secrets=($(load_secrets))
    hab pkg exec core/hab-backline env "${secrets[@]}" bash --login -c "$@";;
```

Reply:
@christophermaier tried this in a previous PR and it doesn't work. That's why we also disable the shellcheck above.